blender/arm/material/cycles_nodes/nodes_shader.py

```diff
@@ -106,7 +106,8 @@ def parse_bsdfprincipled(node: bpy.types.ShaderNodeBsdfPrincipled, out_socket: N
         # clearcoar_normal = c.parse_vector_input(node.inputs[21])
         # tangent = c.parse_vector_input(node.inputs[22])
     if state.parse_opacity:
-        if len(node.inputs) >= 21:
+        state.out_opacity = c.parse_value_input(node.inputs[21])
+        if len(node.inputs) >= 21: #do we need to test that ?
```

Comment:
I think the check is for compatibility with old node versions (Blender doesn't automatically update them as far as I know...)

Reply:
verstanden